content/tr/auto-scaling.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Otomatik Ölçekleme
+status: Completed
+category: property
+tags: ["altyapı", "", ""]
+---
+
+Otomatik ölçekleme, bir sisteme otomatik olarak, genellikle bilişim kaynakları açısından, ihtiyaç duyulduğunda kaynak ekleme yeteneğidir. Otomatik ölçekleme sistemiyle kaynaklar ihtiyaç duyulduğunda otomatik olarak eklenir ve dalgalanan kullanıcı taleplerini karşılayacak şekilde ölçeklenebilir. Otomatik ölçekleme süreci çeşitlilik gösterebilir ve bellek veya işlem süresi gibi farklı ölçütlere dayalı olarak ölçeklenebilir. Yönetilen bulut hizmetleri genellikle otomatik ölçekleme işlevselliği ile ilişkilendirilir çünkü çoğu şirket içi dağıtımdan daha fazla seçenek ve uygulama bulunmaktadır.
+
+Daha önce altyapı ve uygulamalar, yoğun sistem kullanımını dikkate alacak şekilde tasarlanıyordu. Bu mimari, daha fazla kaynağın gereğinden az kullanıldığı ve değişen tüketici talebine karşı esnek olmadığı anlamına geliyordu. Bu esneklik eksikliği, işletmeye daha yüksek maliyetler ve talep fazlasından kaynaklanan kesintiler nedeniyle iş kaybı anlamına geliyordu.
+
+Bulut teknolojisi, uygulamaları ve bağımlılıklarını [sanallaştırma](/virtualization/) ve [konteynerleştirme](/containerization/) yoluyla ölçeklenebilir uygulamalar oluşturarak, organizasyonlara kullanıcı taleplerine göre ölçeklenebilir uygulamalar inşa etme olanağı sağlar. Uygulama taleplerini izleyip otomatik olarak ölçeklenerek, optimal bir kullanıcı deneyimi sunabilirler. Netflix'in her Cuma akşamı yaşadığı izleyici artışını düşünün. Otomatik ölçekleme, dinamik olarak daha fazla kaynak eklemeyi ifade eder: örneğin, daha fazla video akışına izin veren sunucu sayısını artırma ve tüketim normalleştikten sonra ölçeği küçültme.
```

Comment:
Could you update the links to include the `/tr/` prefix, so that they link to the Turkish content?

Reply:
updated and pushed.